python_modules/dagster/dagster/_core/storage/schedules/sql_schedule_storage.py

```diff
@@ -211,9 +225,16 @@ def _jobs_has_selector_state(self, conn, selector_id):
         result = conn.execute(query)
         row = result.fetchone()
         result.close()
-        return row[0] > 0
+        return row[0] > 0  # type: ignore
```

Comment:
how does this not return a bool?

Reply:
the issue there is that `row` is a possible none-- updated commetn